api/files/views.py

```diff
@@ -35,7 +35,249 @@ def get_file(self, check_permissions=True):
+        name          string     name of the new folder
+        path          string     OSF path/id of the file
+        materialized  string     the full path of the folder relative to the storage root
+        kind          string     "file"
+        etag          string     etag - http caching identifier
+        modified      timestamp  last modified timestamp - format depends on provider
+        contentType   string     null if provider="osfstorage", else MIME-type
+        provider      string     id of provider e.g. "osfstorage", "s3", "googledrive"
+        size          integer    size of file in bytes
+        extra         object
+          version     integer    version number of file. will be 1 on initial upload
+          downloads   integer    count of the number times the file has been downloaded
+          hashes      object
+            md5       string     md5 hash of file
+            sha256    string     SHA-256 hash of file
+
+    ####Folder Entity
+
+        name          type    description
+        ----------------------------------------------------------------------
+        name          string  name of the new folder
+        path          string  OSF path/id of the folder
+        materialized  string  the full path of the folder relative to the storage root
+        kind          string  "folder"
+        etag          string  etag - http caching identifier
+        extra         object  null
+
+
+    ##Attributes
+
+    For an OSF File entity, the `type` is "files" regardless of whether the entity is actually a file or folder.  They
+    can be distinguished by the `kind` attribute.  Files and folders use the same representation, but some attributes may
+    be null for one kind but not the other. `size` will be null for folders.  A list of storage provider keys can be
+    found [here](/v2/#storage-providers).
+
+        name          type               description
+        ---------------------------------------------------------------------------------
+        name          string             name of the file or folder; use for display
+        kind          string             "file" or "folder"
+        path          url path           unique path for this entity, used in "move" actions
+        size          integer            size of file in bytes, null for folders
+        provider      string             storage provider for this file. "osfstorage" if stored on the OSF.  Other
+                                         examples include "s3" for Amazon S3, "googledrive" for Google Drive, "box"
+                                         for Box.com.
+        last_touched  iso8601 timestamp  last time the metadata for the file was retrieved. only applies to non-OSF
+                                         storage providers.
+
+    ##Relationships
+
+    ###Checkout (*files, folders*)
+
+    ** *Checkouts are still a work-in-progress. This property may be set but is currently a no-op.* **
```

Comment:
Let's leave checkouts out until it's live. If it's showing up in the schema, can we disable it until it's finalized?